content/200-concepts/100-components/01-prisma-schema/09-index-configuration.mdx

```diff
@@ -14,24 +14,37 @@ From version `3.5.0` onwards, Prisma allows configuration of indexes, unique con
 
 - The `sort` argument is available for all databases on the `@unique`, `@@unique` and `@@index` fields. Additionally, SQL Server also allows it on `@id` and `@@id`.
```

Comment:
```suggestion
- The `sort` argument is available for all databases on the `@unique`, `@@unique` and `@@index` fields. Additionally, SQL Server also allows it on `@id` and `@@id` and MongoDB on `@@fulltext`.
```

Reply:
I've included this but moved it further down with the other 3.6.0 information